### PR TITLE
Ingestion/tests

### PR DESCRIPTION
**Pull request by cfarre**
unit tests with mocks, tests are passing in dev box and in Linux when building the service

### CODE REVIEW

**Comment by cfarre** [comment]
@nithinpnp 
Nithin I talked to Rohit and he said you could review this PR. Basically is all unit tests for the ingestion service using mockito. Our code coverage was lame in the project and I improved it as per management request. thanks

File: src/shipping/ingestion/src/test/java/com/fabrikam/dronedelivery/ingestion/IngestionPerformanceTest.java [hunk 1]
```diff
@@ -1,221 +0,0 @@
-package com.fabrikam.dronedelivery.ingestion;
-
```
Comment (nithinpnp) [inline src/shipping/ingestion/src/test/java/com/fabrikam/dronedelivery/ingestion/IngestionPerformanceTest.java]: Carlos , Why is this getting removed. Do we have any startergy to get this in to some where else ? 
If not , please check for concept called profiles . That should let this code remain and not interfere when we run mvn clean build

**Comment by cfarre** [comment]
@nithinpnp 
I can put back the integration tests. i removed them because they could not be run during a clean build. I was marking them as ignore attribute. can you send me a link to how to set the profile?

**Comment by nithinpnp** [comment]
https://zeroturnaround.com/rebellabs/the-correct-way-to-use-integration-tests-in-your-build-process/

https://www.petrikainulainen.net/programming/maven/integration-testing-with-maven/ -- this talks about using maven profile

**Comment by cfarre** [comment]
@nithinpnp @ferantivero 
Nithin i put back the integration tests plus the profile, so integration tests do not run during a clean build. they will only run on a Jenkins scenario. Please approve when you can

Fernando this has been tested with our deployment scripts. all tests passed

**Comment by nithinpnp** [comment]
approved

**Comment by cfarre** [comment]
@nithinpnp @ferantivero 

can  you merger this PR?

**Comment by nithinpnp** [comment]
Carlos,

I was still not used to the web based version of open Github. I closed the pullrequest by mistakingly after approving . Could you please revert it back

Regards,
Nithin MadhavanPillai

**Comment by nithinpnp** [comment]
I merged it dude.. take care.. no issues.

Regards,
Nithin MadhavanPillai


From: Nithin MadhavanPillai <Nithin.MadhavanPillai@microsoft.com>
Date: Thursday, February 8, 2018 at 1:21 PM
To: mspnp/microservices-reference-implementation <reply@reply.github.com>, mspnp/microservices-reference-implementation <microservices-reference-implementation@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: Re: [mspnp/microservices-reference-implementation] Ingestion/tests (#91)

Carlos,

I was still not used to the web based version of open Github. I closed the pullrequest by mistakingly after approving . Could you please revert it back

Regards,
Nithin MadhavanPillai

**Comment by cfarre** [comment]
@nithinpnp 
thanks much

**Pull request merged**